doc/sources/installation/installation-linux-venvs.rst

```diff
@@ -0,0 +1,349 @@
+    pip install -U pygments docutils
+
+
+.. _linux-run-app:
+
+
+Start from the Command Line
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We ship some examples that are ready-to-run. However, these examples are packaged inside the package.
+This means you must first know where easy_install has installed your current kivy package,
+and then go to the examples directory::
+
+    $ python -c "import pkg_resources; print(pkg_resources.resource_filename('kivy', '../share/kivy-examples'))"
+
+And you should have a path similar to::
+
+    /usr/local/lib/python2.6/dist-packages/Kivy-1.0.4_beta-py2.6-linux-x86_64.egg/share/kivy-examples/
+
+Then you can go to the example directory, and run it::
+
+    # launch touchtracer
+    $ cd <path to kivy-examples>
+    $ cd demo/touchtracer
+    $ python main.py
+
+    # launch pictures
+    $ cd <path to kivy-examples>
+    $ cd demo/pictures
+    $ python main.py
+
+If you are familiar with Unix and symbolic links, you can create a link directly in your home directory
+for easier access. For example:
+
+#. Get the example path from the command line above
+#. Paste into your console::
+
+    $ ln -s <path to kivy-examples> ~/
+
+#. Then, you can access to kivy-examples directly in your home directory::
+
+    $ cd ~/kivy-examples
+
+If you wish to start your Kivy programs as scripts (by typing `./main.py`) or by double-clicking them,
+you will want to define the correct version of Python by linking to it. Something like::
+
+    $ sudo ln -s /usr/bin/python2.7 /usr/bin/kivy
+
+Or, if you are running Kivy inside a virtualenv, link to the Python interpreter for it, like::
+
+    $ sudo ln -s /home/your_username/Envs/kivy/bin/python2.7 /usr/bin/kivy
+
+Then, inside each main.py, add a new first line::
+
+    #!/usr/bin/kivy
+
+NOTE: Beware of Python files stored with Windows-style line endings (CR-LF). Linux will not ignore the <CR>
+and will try to use it as part of the file name. This makes confusing error messages. Convert to Unix line endings.
+
+Device permissions
+~~~~~~~~~~~~~~~~~~
+
+When you app starts, Kivy uses `Mtdev <http://wiki.ubuntu.com/Multitouch>`_ to
+scan for available multi-touch devices to use for input. Access to these
+devices is typically restricted to users or group with the appropriate
+permissions.
+
+If you do not have access to these devices, Kivy will log an error or warning
+specifying these devices, normally something like::
+
+    Permission denied:'/dev/input/eventX'
+
+In order to use these devices, you need to grant the user or group permission.
+This can be done via::
+
+    $ sudo chmod u+r /dev/input/eventX
+
+for the user or::
+
+    $ sudo chmod g+r /dev/input/eventX
+
+for the group. These permissions will only be effective for the duration of
+your current session. A more permanent solution is to add the user to a group
+that has these permissions. For example, in Ubuntu, you can add the user to
+the 'input' group::
+
+    $ sudo adduser $USER input
```

Comment:
I think that's the only solution that should be indicated, as it's the more correct one.

Reply:
I realize it's merely a displacement of the previously existing documentation, didn't know we had that there, but it might be a good time to trim it down.

Reply:
Sure! I removed the other solutions